recommender/pipeline_configs.py

```diff
@@ -126,33 +104,10 @@
     REWARD_LOWER_BOUND,
 )
 
-SE = 128
-UE = 64
+SE = Services2tensorsEmbedder().embedding_dim
```

Comment:
Shouldnt this be the other way around? Embedders getting their dimensions from config I mean

Reply:
Yes and no. Embedders should get  their dimension from config but on the other hand `Services2tensorsEmbedder` class uses NLP engine and language corpus that has some specific dimension of output tensors so it can't be set. Maybe in the next NLP-related PR it could be solved in some way but on the other hand It is not necessary needed.